CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## 3.0.1-BETA21 - 2022-09-13
+## 3.0.1-BETA23 - 2022-09-14
```

Comment:
Don't forget to create a BETA-24 (at least to test than no E2E are broken, and you can and new E2E tests in a second time)